ui/app/pages/settings/index.scss

```diff
@@ -70,12 +70,11 @@
   }
 
   &__sub-header-text {
-    font-style: normal;
-    font-weight: normal;
-    font-size: 20px;
+    @include H4;
```

Comment:
This change is intentionally different because the original font size falls between two typography options (H4 + H3). This results in the "Networks" title in the below screenshots to change from 20px to 18px.

<details>
<summary>Before</summary>
<img src="https://user-images.githubusercontent.com/4448075/88580270-ff547280-d010-11ea-8a18-5f4578c17730.png" />
</details>

<details>
<summary>After</summary>
<img src="https://user-images.githubusercontent.com/4448075/88580342-15623300-d011-11ea-9694-48a626608b87.png" />
</details>